lib/cookies.js

```diff
@@ -0,0 +1,33 @@
+var util = require('./util');
+
+function setCookie(name, value, expiresAt) {
+  var expiresText = '';
+  if (expiresAt) {
+    expiresText = ' expires=' + util.isoToDate(expiresAt).toUTCString() + ';';
```

Comment:
Do we ever care about having it in date format? How about just making the util function isoToUTCString?